src/functions/handleSummary.ts

```diff
@@ -290,7 +290,8 @@ export async function validateStage(
         activityStatus[stage].status.state = state;
 
         if (state !== OrchestratorComponentState.InProgress &&
-            state !== OrchestratorComponentState.NotStarted) {
+            state !== OrchestratorComponentState.NotStarted &&
+            state !== OrchestratorComponentState.Error) {
```

Comment:
In the case where there's an error we do want to still send a message that the current step failed.  I think this check should be moved down to line 330, where if there's an error we send the sendTaskFailure.  This would allow the status to be bubbled up, and a failure would be recognized by the step function